### PR TITLE
bael-4909: Getting Java Version at Runtime

### PR DESCRIPTION
**Pull request by hmdrzsharifi**

### CODE REVIEW

File: core-java-modules/core-java-11-2/src/test/java/com/baeldung/version/VersionUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    @Test
+    public void givenJava_whenUsingRuntime_thenGetVersion() {
+        String expectedVersion = "11";
+        Runtime.Version runtimeVersion = Runtime.version();
+        String version = String.valueOf(runtimeVersion.version().get(0));
+        Assertions.assertThat(version).isEqualTo(expectedVersion);
+    }
+
+    @Test
+    @Disabled("Disabled until Java version is 11")
```
Comment (glmartin) [inline core-java-modules/core-java-11-2/src/test/java/com/baeldung/version/VersionUnitTest.java:19]: Let's change this label to "Only valid for Java 8 and lower."

File: core-java-modules/core-java-11-2/src/test/java/com/baeldung/version/VersionUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            int separator = version.indexOf(".");
+            if (separator != -1) {
+                version = version.substring(0, separator);
+            }
+        }
+        Assertions.assertThat(version).isEqualTo(expectedVersion);
+    }
+
+    @Test
+    @Disabled("Disabled until Java version is 11")
```
Comment (glmartin) [inline core-java-modules/core-java-11-2/src/test/java/com/baeldung/version/VersionUnitTest.java:35]: Let's change this label to "Only valid for Java 8 and lower."

File: core-java-modules/core-java-11-2/src/test/java/com/baeldung/version/VersionUnitTest.java (hunk 1 above)
Comment (hmdrzsharifi) [reply core-java-modules/core-java-11-2/src/test/java/com/baeldung/version/VersionUnitTest.java:19]: Done.

File: core-java-modules/core-java-11-2/src/test/java/com/baeldung/version/VersionUnitTest.java (hunk 2 above)
Comment (hmdrzsharifi) [reply core-java-modules/core-java-11-2/src/test/java/com/baeldung/version/VersionUnitTest.java:35]: Done.

**Pull request merged**